client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -85,6 +95,41 @@ module.exports = React.createClass( {
 		return info;
 	},
 
+	handleDomainVolumeSelection: function( event ) {
+		event.preventDefault();
+		let volume = parseInt( event.target.value );
```

Comment:
`parseInt` uses a different base only when the string starts with a "0". So technically, we're safe here as we're controlling the supplied value. I don't mind adding the radix though.